Fix-Website/app/app.js

```diff
@@ -34,25 +32,44 @@
     app.directive("infoZone", function () {
         return {
             restrict: 'E',
-            templateUrl: "Templates/info.html",
-            controller: function () {
-                this.form = false;
-                this.clickProduct = function () {
-
-                    this.form = !this.form;
-                };
-            },
-            controllerAs: 'info'
+            templateUrl: "Templates/info.txt"
         };
     });
 
-    app.controller('ReferenceController', ['$scope', function ($scope) {
+    /**
+     * Controller for setting up the scope.
+     */
+    app.controller('ReferenceController', ['$scope','$http', function ($scope, $http) {
         $scope.frame = 1;
+        $scope.active = "basic";
+        $scope.activeProduct = "";
         $scope.checkFrame = function (check) {
-            return this.frame === check;
+            return $scope.frame === check;
         };
-        $scope.setFrame = function (newFrame) {
-            this.frame = newFrame;
+        $scope.setFrame = function (newFrame, newActive) {
+            if (newFrame === 3 & $scope.frame != 3) {
+                $scope.activeProduct = $scope.active;
+            }
+            $scope.frame = newFrame;
+            $scope.active = newActive;
         };
+        $http.get('cmdletsfull.json')
+            .then(function (res) {
+                $scope.products = res.data;
+            });
     }]);
+    /*Angular doesn't sort objects, only arrays, so I had to make my own filter.*/
```

Comment:
This comment isn't useful. You explained why you are creating a filter, but don't explain what it actually _does_. How does it work? What does it actually do? What is `cmdlets` supposed to be? What is `cmdletField`?